octobot_trading/data/portfolio_profitability.py

```diff
@@ -125,13 +124,20 @@ async def _update_profitability(self, force_recompute_origin_portfolio=False):
     async def get_average_market_profitability(self):
         await self.get_current_crypto_currencies_values()
 
-        origin_values = [value / self.origin_crypto_currencies_values[currency]
+        origin_values = [value / self._get_origin_value_using_ref_market(currency)
                          for currency, value
                          in self._only_symbol_currency_filter(self.current_crypto_currencies_values).items()
                          if self.origin_crypto_currencies_values[currency] > 0]
 
         return sum(origin_values) / len(origin_values) * 100 - 100 if origin_values else 0
 
+    def _get_origin_value_using_ref_market(self, currency):
+        # preferably compute value in reference market
+        ref_market_pair = merge_currencies(currency, self.reference_market)
+        if ref_market_pair in set(self.origin_crypto_currencies_values.keys()):
+            return self.origin_crypto_currencies_values[ref_market_pair]
```

Comment:
I don't understand why in sometimes we use `currencies` as keys and some other times `pairs`

Reply:
Right, I removed it and it also fixed an issue with initial portfolio value

Reply:
ok :+1: